manifests/c/CreativeTechnology/SoundBlasterCommand/3.5.07.00/CreativeTechnology.SoundBlasterCommand.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: CreativeTechnology.SoundBlasterCommand
+PackageVersion: 3.5.07.00
+PackageLocale: en-US
+Publisher: Creative Technology Ltd.
+PackageName: Sound Blaster Command
+License: Copyright (c) Creative Technology Ltd., 2016-2020. All Rights Reserved.

```

Comment:
A copyright is not a license. Please update this to the actual license terms such as Proprietary, MIT, GPL-2.0, or whatever they are for the package